recipe/meta.yaml

```diff
@@ -12,9 +11,9 @@ source:
 
 build:
   # Skip on Windows until build problems are fixed
-  skip: true  # [win]
-  merge_build_host: true  # [win]
-  number: 0
+  # Checking windows to see if it passes. Uncomment the line if it fails.
+  # skip: true  # [win]
+  number: 1
```

Comment:
Let's handle windows separately; seems to be failing on regular C++ stdlib headers:
```
D:/bld/r-arrow_1721257668194/_h_env/lib/R/include\Rinternals.h:34:11: fatal error: 'cstdio' file not found
   34 | # include <cstdio>
      |           ^~~~~~~~
```

Reply:
Continued in PR: https://github.com/conda-forge/r-arrow-feedstock/pull/84